src/build/bcd/mapper.ts

```diff
@@ -84,6 +84,23 @@ function mapInterfaceLike(
   };
   const methods = filterMapRecord(i.methods?.method, recordMapper);
   const properties = filterMapRecord(i.properties?.property, recordMapper);
+
+  if (i.iterator) {
+    const iteratorKey = i.iterator.async ? "@@asyncIterator" : "@@iterator";
+    const iteratorCompat = mergeCompatStatements(
+      data[iteratorKey] ?? data["values"]
```

Comment:
In practice, pretty much all iterable interfaces have a `values` entry, but only some have `@@iterator` (e.g. [`MediaKeyStatusMap`](https://developer.mozilla.org/en-US/docs/Web/API/MediaKeyStatusMap#browser_compatibility)). So we *could* get away with always using `data["values"]`, but I don't know if that's very clear.

Reply:
Hmmm. I think it's worth filing an issue on BCD. For now let's live with this with a comment about the situation, and perhaps a link to a BCD issue.

Reply:
Looks like there's already such an issue: https://github.com/mdn/browser-compat-data/issues/6367.

I'll link to it in a code comment.